arcgis-ios-sdk-samples/Features/Feature layer query/FLQueryViewController.swift

```diff
@@ -19,74 +19,101 @@ class FLQueryViewController: UIViewController, UISearchBarDelegate {
-    private var featureLayer:AGSFeatureLayer!
+    private var featureTable:AGSServiceFeatureTable?
+    private var featureLayer:AGSFeatureLayer?
     
     private var selectedFeatures = [AGSFeature]()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         //add the source code button item to the right of navigation bar
-        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["FLQueryViewController"]
+        (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["FLQueryViewController"]
         
-        //initialize map with topographic basemap
-        self.map = AGSMap(basemap: .topographic())
-        //assign map to the map view
-        self.mapView.map = self.map
+        // initialize map with topographic basemap
+        let map = AGSMap(basemap: .topographic())
+        // assign map to the map view
+        mapView.map = map
         
-        //create feature table using a url
-        self.featureTable = AGSServiceFeatureTable(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer/2")!)
-        //create feature layer using this feature table
-        self.featureLayer = AGSFeatureLayer(featureTable: self.featureTable)
+        /// The url of a map service layer containing sample census data of United States counties.
+        let statesFeatureTableURL = URL(string: "https://services.arcgis.com/jIL9msH9OI208GCb/arcgis/rest/services/USA_Daytime_Population_2016/FeatureServer/0")!
+        // create feature table using a url
+        let featureTable = AGSServiceFeatureTable(url: statesFeatureTableURL)
+        self.featureTable = featureTable
         
-        //set a new renderer
+        // create feature layer using this feature table
+        let featureLayer = AGSFeatureLayer(featureTable: featureTable)
+        self.featureLayer = featureLayer
+        // show the layer at all scales
+        featureLayer.minScale = 0
+        featureLayer.maxScale = 0
+        
+        // set a new renderer
         let lineSymbol = AGSSimpleLineSymbol(style: .solid, color: .black, width: 1)
         let fillSymbol = AGSSimpleFillSymbol(style: .solid, color: UIColor.yellow.withAlphaComponent(0.5), outline: lineSymbol)
-        self.featureLayer.renderer = AGSSimpleRenderer(symbol: fillSymbol)
+        featureLayer.renderer = AGSSimpleRenderer(symbol: fillSymbol)
+        
+        // add feature layer to the map
+        map.operationalLayers.add(featureLayer)
         
-        //add feature layer to the map
-        self.map.operationalLayers.add(self.featureLayer)
-        //zoom to a custom viewpoint
-        self.mapView.setViewpointCenter(AGSPoint(x: -11e6, y: 5e6, spatialReference: AGSSpatialReference.webMercator()), scale: 9e7, completion: nil)
+        // center the layer
+        mapView.setViewpointCenter(AGSPoint(x: -11e6, y: 5e6, spatialReference: AGSSpatialReference.webMercator()), scale: 9e7)
```

Comment:
I would suggest getting rid of the explicit reference to `AGSSpatialReference` as part of your cleanup.

Reply:
Done. Thanks!!